packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -16,7 +16,10 @@ spec:
       Tag: "eks"
       Path: "/var/log/messages"
       Parser: "syslog"
-  {{- $total_tolerations := concat (.Values.tolerations) (.Values.fluentbit_tolerations) }}
+  {{- if .Values.additionalLoggingSources.eks.fluentbit.mountPath }}
+    mountPath: {{ .Values.additionalLoggingSources.eks.fluentbit.mountPath }}
```

Comment:
So its possible this would break it if user sets it, but also possible this fixes/does something they want. So its a "use at your own risk" setting for hosted k8s right ?

Reply:
Correct